packages/remix-dev/__tests__/cli-test.ts

```diff
@@ -72,4 +123,215 @@ describe("remix cli", () => {
       expect(!!semver.valid(stdout.trim())).toBe(true);
     });
   });
+
+  describe("the create command", () => {
+    let tmpDir = path.join(process.cwd(), ".tmp");
+
+    // this also tests sub directories
+    it("works for examples in the remix repo", async () => {
+      let projectDir = path.join(tmpDir, "example");
+      let { stdout } = await execFile("node", [
+        remix,
+        "create",
+        projectDir,
+        "--template",
+        "basic",
+        "--no-install",
+      ]);
+      expect(stdout.trim()).toBe(
+        `💿 That's it! \`cd\` into "${projectDir}" and check the README for development and deploy instructions!`
+      );
+      expect(fs.existsSync(path.join(projectDir, "package.json"))).toBeTruthy();
+      expect(fs.existsSync(path.join(projectDir, "app/root.tsx"))).toBeTruthy();
+    });
+
+    // TODO: enable once this is live
+    it.skip("works for templates in the remix org", async () => {
+      let projectDir = path.join(tmpDir, "template");
+      let { stdout } = await execFile("node", [
+        remix,
+        "create",
+        projectDir,
+        "--template",
+        "arc-stack",
```

Comment:
FWIW I think this one is going to be called the `grunge-stack`

Reply:
I'll update :) - it does get redirected though 🤷‍♂️